gradle.properties

```diff
@@ -0,0 +1,4 @@
+grpcVersion = 1.46.0
+protobufGradle = 0.8.18
+protobufVersion = 3.20.1
+protocVersion = 3.20.1
```

Comment:
I would move it:
```
// Gradle plugin dependencies
protobufGradlePluginVersion = 0.8.18
// Java dependencies
def grpcVersion = "1.46.0"
def protobufVersion = "3.20.1"
def protocVersion = protobufVersion
```
in `dependencies/build.gradle`
For me, the gradle properties at the root is for all modules and for specific gradle, here it is only useful for the dependencies module

Reply:
Yes, I see the problem with `protobufGradle`  perhaps `ext` section in the `dependencies/build.gradle` instead:
```
ext {
  // Gradle plugin dependencies
  protobufGradlePluginVersion = "0.8.18"
}
```

Reply:
I was able to put the grpc and protobuf versions inside a `build.gradle` at the root of the project. This way, each subproject can inherit them. A build.gradle at the root should be following best practices in writing a grade project (I saw it in my work experience).

The only thing I was not able to do is to have `protobufGradlePluginVersion` as well in this new file `build.gradle`. The reason is that this version is the one used by the plugin `"com.google.protobuf"`, and the version of a plugin does not work with `ext`. 

For the moment I did not find better than this.

Remark: I added as well a pluginManagement inside `settings.gradle`, to control the plugin dependencies. I think this is also a good practice.